resources/analysisTools/copyNumberEstimationWorkflow/conda.sh

```diff
@@ -1,6 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Why is this file gone?

Reply:
It was moved but the original somehow got stuck here. The new position is together with the other environment scripts in "copyNumberEstimationWorkflows/environments/" subdirectory.